Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Card.css';
+
+
+const CardContent = () => {
+    return (
+        <div className="cardText">
+            <h2>Get started with React</h2>
+            <p>React makes it painless to create interactive UIs.
+                Design simple views for each state in your application.
+            </p>
+           
+            <a href="http://reactjs.com" onClick={this.handleClick}>reactjs.com</a>
```

Comment:
You don't have `this.handleClick` on CardContent and this cause your app to break. Remove this `onClick` prop and the app will run again.